infrastructure/terraform/ingress.yaml

```diff
@@ -36,10 +36,10 @@ spec:
             - {service_name: queue, project_name: taskcluster-queue}
             - {service_name: hooks, project_name: taskcluster-hooks}
             - {service_name: index, project_name: taskcluster-index}
-            - {service_name: gce-provider, project_name: gce-provider}
```

Comment:
❗️

Reply:
Haha, as discussed in irc, this was a crappy prototype that I made.